packages/schema/docs/build/schema.md

```diff
@@ -145,6 +147,82 @@ Key | Required | Type | Description
 
 -----
 
+## /AuthFieldSchema
+
+Field schema specialized for authentication fields.
+
+#### Details
+
+* **Type** - `object`
+* [**Source Code**](https://github.com/zapier/zapier-platform/blob/zapier-platform-schema@16.3.0/packages/schema/lib/schemas/AuthFieldSchema.js)
+
+#### Properties
+
+Key | Required | Type | Description
+--- | -------- | ---- | -----------
+`key` | **yes** | `string` | A unique machine readable key for this value (IE: "fname").
+`isNotSecret` | no | `boolean` | Indicates if this authentication field is safe (not secret).
+`label` | no | `string` | A human readable label for this value (IE: "First Name").
+`helpText` | no | `string` | A human readable description of this value (IE: "The first part of a full name."). You can use Markdown.
+`type` | no | `string` in (`'string'`, `'number'`, `'boolean'`, `'datetime'`, `'password'`, `'copy'`) | The type of this value. Field type of `file` will accept either a file object or a string. If a URL is provided in the string, Zapier will automatically make a GET for that file. Otherwise, a .txt file will be generated.
```

Comment:
The description refers to `file` even though that is no listed as a valid value:

```suggestion
`type` | no | `string` in (`'string'`, `'number'`, `'boolean'`, `'datetime'`, `'password'`, `'copy'`) | The type of this value.
```